lib/broccoli/ember-app.js

```diff
@@ -194,7 +194,7 @@ class EmberApp {
     @param {Object} options
   */
   _initTestsAndHinting(options) {
-    let testsEnabledDefault = process.env.EMBER_CLI_TEST_COMMAND || !this.isProduction;
+    let testsEnabledDefault = (process.env.EMBER_CLI_TEST_COMMAND && JSON.parse(process.env.EMBER_CLI_TEST_COMMAND)) || !this.isProduction;
```

Comment:
We don't need to parse the string (its only ever set to `true`), we should probably just coerce to boolean. 

Mind updating to:

```js
let testsEnabledDefault = process.env.EMBER_CLI_TEST_COMMAND === 'true' || !this.isProduction;
```